config/nav.yml

```diff
@@ -8,9 +8,9 @@ nav:
 ###############################################################################
     # Getting started
 ###############################################################################
-    - Tutorial:
-        - Tutorial introduction: getting-started/README.md
-        - Install Knative using quickstart: getting-started/quickstart-install.md
+    - Quickstart:
```

Comment:
This "tutorial" can't be completed without using the Quickstart plugin, so to me it doesn't make sense to name it this way. Getting started doesn't make sense to me either, because it's too generic. Calling it Quickstart gives users a better idea of exactly what they can expect from this section, IMO.

We could have another Tutorials section that provides tutorials that work on any setup and don't require this plugin. In fact, I'd even vote to change https://knative.dev/docs/samples/ to be named Tutorials, and do some proper content editing etc to make those "samples" full, usable tutorials instead.

@csantanapr is there a reason you know of that we need to or would want to keep this named Tutorial, over something like Quickstart? FWIW, Getting started or Quickstart seem to be more common and useful titles:

- https://kubernetes.io/docs/setup/ (Getting started)
- https://docs.github.com/en/get-started/quickstart (Quickstart)
- https://aws.amazon.com/quickstart/?solutions-all.sort-by=item.additionalFields.sortDate&solutions-all.sort-order=desc&awsf.filter-content-type=*all&awsf.filter-tech-category=*all&awsf.filter-industry=*all (AWS Quick Starts)

Reply:
@abrennan89 it was named "Getting Started" and as part of the clean up @snneji change it to "Tutorial" in this PR https://github.com/knative/docs/pull/4688 it was discussed in WG meetings.

Reply:
So I was wrong on both the date and the location.... :laughing:

Reply:
Even Getting Started to me is better than Tutorial tbh. I really dislike this since I don't think it is reflective of the actual content involved.

Reply:
Hi @abrennan89. We decided against the names "Getting started" and "Quickstart" because they imply that this is where you must start. However, you do not need to do this section to get started with Knative.

Users were confused by these names because they thought that this section would help them to install Knative to do their own project, similar to the GitHub, Kubernetes, and AWS docs you linked above, but they found out that it contains an example that they didn't need to follow. With the name "Tutorial" at least users know they can skip it if they just want to install Knative. 

If you can come up with a better name that also conveys that this section is optional, I'd be happy to for you to change it.